doc/api/http.md

```diff
@@ -1974,7 +1974,7 @@ added: v0.1.90
 Request URL string. This contains only the URL that is
 present in the actual HTTP request. If the request is:
 
-```txt
+```text
```

Comment:
```suggestion
```http
```

Reply:
`http`!? TIL

Reply:
I know, me too! I didn't even think to include that in the bundle. >.<